src/Performance/Time/TimerIsAlreadyStartedException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Performance\Time;
+
+/**
+ * @internal
+ */
+final class TimerIsAlreadyStartedException extends \Exception
```

Comment:
I would rather extend `\LogicException`: this would be a wrong usage not an exception you would like to catch